cfgov/unprocessed/js/modules/util/dom-traverse.js

```diff
@@ -48,23 +47,33 @@ function not( elems, exclude ) {
  * Get the nearest parent node of an element.
  *
  * @param {HTMLNode} elem - A DOM element.
- * @param {string} className - CSS classname.
- * @returns {HTMLNode} Nearest parent node that contains className, or null.
+ * @param {string} selector - CSS selector.
+ * @returns {HTMLNode} Nearest parent node that matches the selector.
  */
-function closest( elem, className ) {
+function closest( elem, selector ) {
   elem = elem.parentNode;
-  do {
-    if ( elem.classList && elem.classList.contains( className ) ) {
-      return elem;
+
+  var matchesSelector = elem.matches ||
+                        elem.webkitMatchesSelector ||
+                        elem.mozMatchesSelector ||
+                        elem.msMatchesSelector;
+  var match;
+
+  while ( elem ) {
+    if ( !matchesSelector.bind( elem )( selector ) ) {
+      elem = elem.parentElement;
+    } else {
+      match = elem;
     }
-    elem = elem.parentNode;
-  } while ( elem );
+
+    if ( match ) return elem;
+  }
 
   return null;
 }
 
 module.exports = {
-  closest:     closest,
   getSiblings: getSiblings,
-  not:         not
+  not:         not,
+  closest:     closest
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/f217937d4eec3537f28765b2f956d29b1609df50